Active Documents/contenteditable-disabled.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
```

Comment:
Should this be renamed to match the original file name so we can better diff them?

Reply:
I think that would make it easier to see what exactly has changed, yes.

Reply:
sounds good, will do

Reply:
That shouldn't matter... git should have picked up that this was a file rename, not a delete, and done the right thing :(

Reply:
(as a later thing to do later, we need to run HTML 5 tidy or prettier over this doc... will also help with diffing)

Reply:
>That shouldn't matter... git should have picked up that this was a file rename, not a delete, and done the right thing :(

Not when the file changes too much 👀